include/ibamr/RelaxationLSMethod.h

```diff
@@ -152,18 +157,29 @@ class RelaxationLSMethod : public IBAMR::LSInitStrategy
     /*!
      * \brief Do one relaxation step over the hierarchy.
      */
-    void relax(SAMRAI::tbox::Pointer<IBTK::HierarchyMathOps> hier_math_ops,
+    void relax(SAMRAI::tbox::Pointer<IBTK::HierarchyGhostCellInterpolation> D_fill_op,
+               SAMRAI::tbox::Pointer<IBTK::HierarchyMathOps> hier_math_ops,
                int dist_idx,
                int dist_init_idx,
-               const int iter) const;
+               int dist_copy_idx,
+               const int iter,
+               const double time) const;
```

Comment:
This isn't backwards-compatible - can you add a stub and deprecate it?

Reply:
Is this about the `relax()` routine. This is private function, so it should not affect the compatibility, right?

Reply:
I think it should be fine to make these kinds of changes to the private interface.

Reply:
My mistake - I didn't see the `private:` label!